authors/apps/authentication/models.py

```diff
@@ -116,5 +116,11 @@ def get_short_name(self):
         the user's real name, we return their username instead.
         """
         return self.username
-
-
+    
+    def token(self):
```

Comment:
Hey Miriam is it possible to make this modular so that we can be able to set different expiry periods for the token

Reply:
Thanks Simon. Let me modify it.